en/docs/deploy-and-publish/deploy-on-gateway/api-microgateway/deploy-api/deploy-api-to-microgateway.md

```diff
@@ -0,0 +1,146 @@
+# Deploy an API to Microgateway
+
+There are two ways to add an API to the Microgateway.
+
+1. [Via API Manager](#via-api-manager)
+
+    By enabling the Control Plane Event Hub in the Microgateway configuration file, the APIs deployed in API Manager will automatically be available in Microgateway. 
+
+2. [Via the CLI tool (apictl)](#via-the-cli-tool-apictl)
+
+    If you are not using WSO2 API Manager yet, or only want to tryout microgateway, simply use the CLI tool [apictl](https://github.com/wso2/product-apim-tooling/releases) to host your API.
+
+## Via API Manager
+
+!!! info
+    **Before you begin**
+
+    - Make sure you have installed docker and docker-compose on your machine.
+
+    - Download the latest [Microgateway 4.0.0 release](https://github.com/wso2/product-microgateway/releases)and extract it to a folder of your choice. The extracted folder will be referred to as `MG_HOME` here onwards.
+
+    - This guide assumes that you have already started the WSO2 API Manager instance. If not, download the latest [release](https://github.com/wso2/product-apim/releases)and follow the steps [here](https://github.com/wso2/product-apim#installation--running)
```

Comment:
This is ok currently but this needs to be updated once the GA release goes out.

Reply:
Will do once the GA is released